DuckDuckGo/Tab/UserScripts/IdentityTheftRestorationPagesUserScript.swift

```diff
@@ -98,4 +105,13 @@ final class IdentityTheftRestorationPagesFeature: Subfeature {
             return [String: String]()
         }
     }
+
+    func getFeatureConfig(params: Any, original: WKScriptMessage) async throws -> Encodable? {
+        [PrivacyProSubfeature.useUnifiedFeedback.rawValue: subscriptionFeatureAvailability.usesUnifiedFeedbackForm]
+    }
+
+    func openSendFeedbackModal(params: Any, original: WKScriptMessage) async throws -> Encodable? {
+        NotificationCenter.default.post(name: .OpenUnifiedFeedbackForm, object: nil, userInfo: ["source": "itr"])
```

Comment:
💡 As I mentioned above, we should leverage the current `UnifiedFeedbackSource` enum to send this information.